prow/pjutil/filter_test.go

```diff
@@ -149,7 +150,8 @@ func TestCommandFilter(t *testing.T) {
 		{
 			name: "truncate-name",
 			body: `/test trigger
-fill in random content so that it exceeds the limit of 50 chars`,
+fill in random content so that it exceeds the limit of 70 chars`,
```

Comment:
nit: "...exceeds the limit of half*2 chars"?

Reply:
ack and fixed